ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -84,16 +87,48 @@ static boolean bug1602844Affected(Map<String, String> attributes) {
    * Redact fields that may contain unintended sensitive information, replacing with null or
    * other appropriate signifiers.
    */
-  public static void redact(Map<String, String> attributes, ObjectNode json) {
+  public static void redact(Map<String, String> attributes, ObjectNode json)
+      throws MessageShouldBeDroppedException, AffectedByBugException {
     if (bug1602844Affected(attributes)) {
       json.path("events").elements().forEachRemaining(event -> {
         JsonNode eventMapValues = event.path(5);
         if (eventMapValues.has("fxauid")) {
           ((ObjectNode) eventMapValues).replace("fxauid", NullNode.getInstance());
-          countRedactedBug1602844.inc();
+        }
+
+        try {
+          handleBug("1602844", ScrubAction.REDACT);
+        } catch (Exception e) {
+          // redactions don't throw exceptions, so this is just ignored
         }
       });
     }
   }
 
+  /**
+   * Inspect the contents of the payload and return true if the content matches a known pattern
+   * we want to scrub the message and write to errors.
+   */
+  public static void writeToErrors(Map<String, String> attributes, ObjectNode json)
+      throws MessageShouldBeDroppedException, AffectedByBugException {
+    if (ParseUri.TELEMETRY.equals(attributes.get(Attribute.DOCUMENT_NAMESPACE)) //
```

Comment:
For Glean pings, the document namespace will not be "telemetry"; rather, it is generally the application ID. As merged here, this PR supports removing the coffee id from desktop telemetry and from core pings, but we are not catching coffee IDs for Glean.